src/instructions/Instructions.test.jsx

```diff
@@ -75,6 +79,62 @@ describe('SequenceExamWrapper', () => {
     expect(getByTestId('sequence-content')).toHaveTextContent('Sequence');
   });
 
+  it('Shows onboarding exam entrance instructions when receives onboarding exam', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        verification: {
+          status: 'approved',
+        },
+        proctoringSettings: {},
+        activeAttempt: {},
+        exam: {
+          time_limit_mins: 30,
+          type: ExamType.ONBOARDING,
+          attempt: {},
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div data-testid="sequence-content">Sequence</div>
```

Comment:
We can remove this attribute `data-testid="sequence-content"` because we don't use it.